src/program/render.rs

```diff
@@ -384,3 +384,26 @@ fn help_text() -> Vec<&'static str> {
         "+------------+----------------------------+-----------------------------------------------------+",
     ]
 }
+
+fn command_help_text() -> Vec<&'static str> {
+    vec![
+        "+-------+------------+----------+-------------------------------------+",
+        "| Short | Long       | Argument | Action                              |",
+        "+-------+------------+----------+-------------------------------------+",
+        "| ng    | newglob    | Required | The new glob/directory/file         |",
+        "| ?     | help       | None     | Toggle help box                     |",
```

Comment:
On an invalid command. [Let the user know that help is available.](https://github.com/Davejkane/riv/blob/029464844ccab490840851af2cb784ff5a524848/src/program/command_mode.rs#L65)

Maybe something like
`No such command "{}". Type "?" for help`